tac/project-updates/2024/2024-Q4-Hyperledger-Solang.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+title: 2024 Q4 Hyperledger Solang
+parent: 2024
+grand_parent: Project Updates
+---
+
+# Project Health
+
+* Hyperledger Solang development is going well, with the main focus on the Soroban target.
+
+* Maintainers Sean Young and Salaheldin met at [Stellar's Meridian 2024 conference](https://meridian.stellar.org/), where Salaheldin gave a talk on Solang. The conference was a great opportunity to attract new contributors to the project.
+
+* Solang Playground Deliverables to the Web3 Foundation were completed and delivered, making the playground closer to the final version.
+
+# Questions/Issues for the TOC
+
+There are no issues at this time.
+
+# Releases
+
+* Cross contract calls and value transfers are not yet merged into the Solang's main, but a release will be made once they are.
+
+# Overall Activity in the Past Quarter
+
+## For Solang
+
+* Worked on cross-contract calls and value transfers for Soroban target.
+
+* Worked on [Soroban's auth framework](https://github.com/stellar/stellar-protocol/blob/master/core/cap-0046-11.md), which is to be demonstrated at the end of Q1 2025. 
+
+* Worked on new Solidity builtins for Soroban target.
+
+* Worked on Soroban's more complex data types (i128, addresses, etc.).
+
+# Current Plans
+
+* Merge cross-contract calls and value transfers into the main branch.
+* Complete Soroban's auth framework.
+
+# Maintainer Diversity
+
+Two new maintainers were added to the project: [Ahmad Sameh](https://github.com/ahmadsamehh) and [Tarek Nasser](https://github.com/tareknaser).
```

Comment:
It would be useful to know what the distribution/breakdown of current maintainers is i.e. how many and from how many orgs

Reply:
Hello @tkuhrt @matthew1001 
I have misused the word `maintainers`. Tarek and Ahmad are usual contributors rather than maintainers, and they don't represent organizations